tests/acceptance/news-test.js

```diff
@@ -0,0 +1,21 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+
+module('Acceptance | news', function(hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /actualites', async function(assert) {
+    //when
+    await visit('/actualites');
+    //then
+    assert.equal(currentURL(), '/actualites');
+  });
+
+  test('it should redirect to index', async function(assert) {
```

Comment:
Le nom de tes tests ne reflètent pas forcément ton besoin métier. Je verrais plutôt quelque chose comme =
`visiting /actualites` -> `it should visit the requested page when the url is valid`
`it should redirect to index` -> `it should redirect to index when the url is invalid`

Et là, on peut aussi envisager des contextes
`when the url is valid`
------ `it should visit the requested page`

`when the url is invalid`
------ `it should redirect to the index`